RadarSDK/RadarVerificationManager.m

```diff
@@ -153,105 +162,28 @@ - (void)trackVerifiedWithBeacons:(BOOL)beacons completionHandler:(RadarTrackComp
     }];
 }
 
-- (void)trackVerifiedTokenWithCompletionHandler:(RadarTrackTokenCompletionHandler)completionHandler {
-    [self trackVerifiedTokenWithBeacons:NO completionHandler:completionHandler];
-}
-
-- (void)trackVerifiedTokenWithBeacons:(BOOL)beacons completionHandler:(RadarTrackTokenCompletionHandler)completionHandler {
-    [[RadarAPIClient sharedInstance]
-     getConfigForUsage:@"verify"
-     verified:YES
-     completionHandler:^(RadarStatus status, RadarConfig *_Nullable config) {
-        [[RadarLocationManager sharedInstance]
-         getLocationWithDesiredAccuracy:RadarTrackingOptionsDesiredAccuracyHigh
-         completionHandler:^(RadarStatus status, CLLocation *_Nullable location, BOOL stopped) {
-            if (status != RadarStatusSuccess) {
-                if (completionHandler) {
-                    [RadarUtils runOnMainThread:^{
-                        completionHandler(status, nil);
-                    }];
-                }
-                
-                return;
+- (void)startTrackingVerifiedWithInterval:(NSTimeInterval)interval beacons:(BOOL)beacons {
+    self.started = YES;
+    
+    __block void (^trackVerified)(void);
+    __block __weak void (^weakTrackVerified)(void);
+    trackVerified = ^{
+        weakTrackVerified = trackVerified;
+        
```

Comment:
This pattern of using a `weak` version is interesting (I wanna read more up on this)

Reply:
Same pattern as `mockTracking`, still need to test here https://github.com/radarlabs/radar-sdk-ios/blob/master/RadarSDK/Radar.m#L324